backend/api/views.py

```
@@ -367,6 +367,64 @@
             )
 
 
+class WebcamUploadView(APIView):
+    """Generate SAS upload URL for webcam recordings."""
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request, video_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `post`

(ANN201)